packages/story-editor/src/storyEditor.js

```diff
@@ -57,7 +58,12 @@ import KeyboardOnlyOutlines from './utils/keyboardOnlyOutline';
 import defaultConfig from './defaultConfig';
 
 function StoryEditor({ config, initialEdits, children }) {
-  const _config = useMemo(() => deepMerge(defaultConfig, config), [config]);
+  // Need `cloneDeep` so that karma tests are not using a modified
+  // `defaultConfig` between runs in CI
+  const _config = useMemo(
+    () => deepMerge(cloneDeep(defaultConfig), config),
+    [config]
+  );
   const { storyId, isRTL, flags } = _config;
```

Comment:
While this is getting eyes, I'm curious if there's a specific reason `_config` is getting memoized. Pretty sure this is only ever getting run on initial app mount.

Reply:
Yeah, good call. I don't think it needs to be memoized since `config` doesn't change.